### PR TITLE
Add missing rarity trait to AncestryRollOptions

### PR DESCRIPTION
**Pull request by reyzor1991**
Need to fix Adopted Ancestry rule element

### CODE REVIEW

**Comment by reyzor1991** [comment]
squash commits please

File: src/module/item/ancestry/document.ts [hunk 1]
```diff
@@ -143,6 +147,11 @@ class AncestryPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends
             actor.rollOptions.all[`self:trait:${trait}`] = true;
         }
     }
+
+    /** Generate a list of strings for use in predication */
+    override getRollOptions(): string[] {
+        return [...super.getRollOptions(), `item:rarity:${this.rarity}`];
+    }
```
Comment (stwlam) [inline src/module/item/ancestry/document.ts:154]: This needs to accept the standard parameter

Comment (reyzor1991) [reply src/module/item/ancestry/document.ts]: Prefix was added

Comment (stwlam) [reply src/module/item/ancestry/document.ts]: the roll option must then use that prefix

Comment (reyzor1991) [reply src/module/item/ancestry/document.ts]: new version of code 
`return [...super.getRollOptions(prefix), `item:rarity:${this.rarity}`];`

File: src/module/item/ancestry/document.ts [hunk 2]
```diff
@@ -143,6 +147,11 @@ class AncestryPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends
             actor.rollOptions.all[`self:trait:${trait}`] = true;
         }
     }
+
+    /** Generate a list of strings for use in predication */
+    override getRollOptions(prefix = this.type): string[] {
+        return [...super.getRollOptions(prefix), `item:rarity:${this.rarity}`];
```
Comment (stwlam) [inline src/module/item/ancestry/document.ts:153]:
```suggestion
        return [...super.getRollOptions(prefix), `${prefix}:rarity:${this.rarity}`];
```

**Pull request merged**